### PR TITLE
feat(gh): modify issues template to use forms

### PR DESCRIPTION
**Pull request by muryoh**
And redirect feature requests to discussions

### CODE REVIEW

**Comment by muryoh** [comment]
Needs discussions to be enabled :pray:

**Comment by gqio** [comment]
> Needs discussions to be enabled 🙏

discussions enabled

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    id: browsers
+    attributes:
+      label: What browser(s) are you seeing the problem on?
+      description: It may help us identify the problem
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
```
Comment (gqio) [inline .github/ISSUE_TEMPLATE/bug_report.yml:50]: Maybe a `other and specify` ?

Comment (gqio) [reply .github/ISSUE_TEMPLATE/bug_report.yml:50]: at some point we will need the version.

**Review by gqio — APPROVED** [review]

**Comment by gqio** [comment]
> > discussions enabled
> 
> My hero 🤟
> Can you also remove all categories _but_ Ideas? 🙇

done

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (muryoh) [reply .github/ISSUE_TEMPLATE/bug_report.yml:50]: Good point, already merged as-is on backlight, will update it when updating backlight

**Pull request merged**